internal/info/info.go

```diff
@@ -319,9 +320,11 @@ func (i *info) prepare() {
 		if len(i.detail.BuildCPUInfoFlags) == 0 && len(BuildCPUInfoFlags) != 0 {
 			i.detail.BuildCPUInfoFlags = strings.Split(strings.TrimSpace(BuildCPUInfoFlags), " ")
 		}
+		i.baseURL = "https://" + valdRepo + "/tree/" + i.detail.GitCommit
+		i.goRoot = runtime.GOROOT()
```

Comment:
Each of them it's better to check length if length is not 0 we shouldn't overwrite variable.